core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1323,8 +1335,59 @@
         </xs:element>
       </xs:choice>
     </xs:sequence>
+    <xs:attribute name="max-size" type="xs:string">
+      <xs:annotation>
+        <xs:documentation>
+           Defines the size of the data container in a byte quantity plus an optional multiplier.
+           Supported multipliers are kB (kilobytes), MB (megabytes), GB (gigabytes) and TB (terabytes).
+           Eviction occurs either the amount of memory exceeds the maximum size.
```

Comment:
```suggestion
           Eviction occurs after the approximate memory usage exceeds the maximum size.
```

Reply:
Guessing we may also want to mention that binary or off heap media type is specified.